static/js/robot.js

```diff
@@ -401,6 +416,18 @@ class RobotClass {
             dir = (this.dir + theta*180/Math.PI) % 360;
         }
 
+        if (onRamp != null) {
+            if (onRamp.highSide == 90.0) {
+                dy = dy + onRamp.incline * 0.01;
```

Comment:
I think we may want to increase the offset, it feels a little weaker than it should be

Reply:
Also maybe make the multiplier a constant value defined just above these lines